packages/components/src/RichTooltip/RichTooltip.scss

```diff
@@ -0,0 +1,110 @@
+$tc-popover-border-width: 0.1rem !default;
+$tc-popover-border-color: rgb(226, 226, 226) !default;
+$tc-popover-box-shadow: 0 0.1rem 0.3rem 0 rgba(0, 0, 0, 0.2) !default;
+$tc-popover-body-min-height: 10rem !default;
+$tc-popover-body-max-height: 30rem !default;
+$tc-popover-max-width: 40rem !default;
+$tc-popover-header-border-color: #ededed !default;
+$tc-popover-header-footer-height: 5rem !default;
+$tc-popover-header-font-size: 1.6rem !default;
+$tc-popover-header-font-weight: 600 !default;
+$tc-popover-icon-error-size: 3.6rem !default;
+
+:global(.popover) {
+	padding: 0;
+	border: $tc-popover-border-width solid $tc-popover-border-color;
+	border-radius: $border-radius-base;
+	background-color: $white;
+	box-shadow: $tc-popover-box-shadow;
+	max-width: $tc-popover-max-width;
+}
+
+:global(.popover-content) {
+	padding: 0;
+}
+
+.tc-tooltip-header {
+	min-width: $tc-popover-max-width;
+	border-bottom: $tc-popover-border-width solid $tc-popover-header-border-color;
+	padding: 0 $padding-large;
+	flex: 1;
+	height: $tc-popover-header-footer-height;
+	display: flex;
+	align-items: center;
+
+	h1 {
```

Comment:
h1 is for a title of a page.
For example bootstrap3 modal title use h4 and bootstrap 4 use h5.